storybook/src/components/skiplink/skiplink.core-template.ts

```diff
@@ -0,0 +1,13 @@
+import { Skiplink } from "dso-toolkit";
+import { html } from "lit-html";
+
+import { ComponentImplementation } from "../../templates";
+
+export const coreSkiplink: ComponentImplementation<Skiplink> = {
+  component: "skiplink",
+  implementation: "core",
+  template: () =>
+    function skiplinkTemplate({}) {
+      return html`<dso-skiplink to="inhoud" label="Ga naar inhoud"></dso-skiplink>`;
```

Comment:
De properties `to` en `label` nog aansluiten op het Skiplink model

Reply:
Skiplink model geïmplementeerd.